client/src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { ThemeProvider } from "@material-ui/styles";
+
+import Home from "./Home/Home";
+import Landing from "./Landing/Landing";
+import theme from "../styles/Theme";
```

Comment:
I like to avoid relative paths if they are going backwards - check out https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d